dessia_common/workflow/core.py

```diff
@@ -1334,7 +1334,9 @@ def to_script(self) -> str:
         script_imports = self_script.imports_to_str()
 
         return f"{script_imports}\n" \
-               f"{self_script.declaration}"
+               f"{self_script.declaration}\n" \
+               f'documentation = """{self.documentation}"""' \
```

Comment:
I think we should avoid using triple quote inside a plain script.
Consider `f"documentation = '{self.documentation}'"` instead

Reply:
![image](https://github.com/Dessia-tech/dessia_common/assets/109952534/74f3393b-b4cd-4ceb-8c4f-f591c887dd55)

with using `f"documentation = '{self.documentation}'"` :
![image](https://github.com/Dessia-tech/dessia_common/assets/109952534/84c98caa-0e27-4fa4-ad9c-b62155435f5c)


![image](https://github.com/Dessia-tech/dessia_common/assets/109952534/6541c9fe-2a2d-4811-a995-9497579f6ea1)